test/IRGen/autolink-coff.swift

```diff
@@ -15,3 +24,10 @@ import module
 // CHECK-GNU-ASM: .section .drectve
 // CHECK-GNU-ASM: .ascii " -lmodule"
 
+// CHECK-MSVC-IR: !{{[0-9]+}} = !{i32 {{[0-9]+}}, !"Linker Options", [[NODE:![0-9]+]]}
+// CHECK-MSVC-IR: [[NODE]] = !{[[LIST:![0-9]+]]}
+// CHECK-MSVC-IR: [[LIST]] = !{!"/DEFAULTLIB:module.lib"}
+
+// CHECK-MSVC-ASM: .section .drectve
+// CHECK-MSVC-ASM: .ascii " /DEFAULTLIB:module.lib"
+
```

Comment:
Up to you: if you think the PS4 patch is going in soon, you can fold it in here; otherwise I'd prefer to make it separate.